test/server/cards/02_SHD/units/KihraxzHeavyFighter.spec.ts

```diff
@@ -0,0 +1,38 @@
+describe('Kihraxz Heavy Fighter', function () {
+    integration(function (contextRef) {
+        it('should gain +3 on attack if you exhaust another friendly unit and self is not selectable', function () {
```

Comment:
nit: this and the Razor Crest tests are missing the `describe(). Even if the setupTest is in the individual cases, we still include the prefix "Kihraxz Heavy Fighter's ability"